db/migrate/20200709024010_create_gemini_connection.rb

```diff
@@ -0,0 +1,25 @@
+class CreateGeminiConnection < ActiveRecord::Migration[6.0]
+  def change
+    create_table :gemini_connections, id: :uuid, default: -> { "uuid_generate_v4()"}, force: :cascade do |t|
+      t.belongs_to :publisher, type: :uuid, index: true, null: false
+
+      t.string :encrypted_access_token
+      t.string :encrypted_access_token_iv, index: { unique: true }
+
+      t.string :encrypted_refresh_token
+      t.string :encrypted_refresh_token_iv, index: { unique: true }
+
+      t.string :expires_in
+      t.datetime :access_expiration_time
+      t.string :gemini_user_id
+      t.string :display_name
+      t.string :state_token
+
+      t.string :scope
+
+      t.string :status
+      t.string :country
+      t.boolean :is_verified
```

Comment:
May be worth putting an additional index on `is_verified` or `status` ,  whichever one is the most frequently queried on

This could probably go for the Uphold Connection too  🔮

Reply:
Done! :D https://github.com/brave-intl/publishers/pull/2801/commits/5066ce084bc244e44167600d504e4d81f1401b9a